### PR TITLE
[CLEANUP] Remove view options in routes

### PR DESCRIPTION
**Pull request by chadhietala**
This removes the `viewName` and `view` option in `Route`. This is dependent on #13580  or I can make them part of this PR.

### CODE REVIEW

File: packages/ember-routing/lib/system/route.js [hunk 1]
```diff
@@ -2150,7 +2117,7 @@ function buildRenderOptions(route, namePassed, isDefaultRender, _name, options)
     controller.set('model', options.model);
   }
 
-  let viewName = options && options.view || namePassed && name || route.viewName || name;
+  let viewName = namePassed && name || name;
   let ViewClass = owner._lookupFactory(`view:${viewName}`);
```
Comment (chadhietala) [inline packages/ember-routing/lib/system/route.js]: I'm assuming this cannot be removed because Outlets are implemented and registered as `View`?

Comment (rwjblue) [inline packages/ember-routing/lib/system/route.js]: I believe the whole thing here can be removed.  We shouldn't need any `view:*` lookups here...

**Comment by rwjblue** [comment]
This looks good, can you squash and rebase now that #13626 has landed?

**Comment by rwjblue** [comment]
Rebased and submitted #13642.

**Pull request closed without merging**